src/Language/buildSchema.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Digia\GraphQL\Language;
+
+use Digia\GraphQL\Language\AST\Node\DocumentNode;
+use Digia\GraphQL\Language\AST\Node\NamedTypeNode;
+use Digia\GraphQL\Language\AST\Schema\DefinitionBuilder;
+use Digia\GraphQL\Language\AST\Schema\SchemaBuilder;
+use Digia\GraphQL\Type\Contract\SchemaInterface;
+
+/**
+ * @param string $source
+ * @param array  $options
+ * @return SchemaInterface
+ * @throws \Digia\GraphQL\Error\GraphQLError
+ * @throws \Exception
+ * @throws \TypeError
+ */
+function buildSchema(string $source, array $options = []): SchemaInterface
```

Comment:
Should it be `BuildSchema` for consistency?

Reply:
I mean since we have `GraphQLSchema`, `GraphQLXXX` function. In Laravel functions should be in snake_case :-?

Reply:
Usually, functions are named using camelCase in PHP. GraphQLXXX is an exception to this, which I made because it makes it look more consistent with the other implementations.